packages/date/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@contentful/field-editor-date",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
The changes in the date package should be done in `master`. Use `next` only for rich-text changes